<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.13.0 on win32)" generated="2024-11-21T10:52:50.220568" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\M4800\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-11-21T10:52:50.261934" elapsed="6.294241"/>
</kw>
<status status="PASS" start="2024-11-21T10:52:50.261202" elapsed="6.295207"/>
</test>
<status status="PASS" start="2024-11-21T10:52:50.222249" elapsed="6.334980"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-11-21T10:52:50.194405" level="WARN">Error in file 'c:\Users\M4800\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
